web/annotation_layer_builder.css

```diff
@@ -13,6 +13,10 @@
  * limitations under the License.
  */
 
+:root {
+  --unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
```

Comment:
This seem to be using the exact variable name as in https://github.com/mozilla/pdf.js/blob/master/web/xfa_layer_builder.css, could that possibly become an issue?

Given that all CSS files are bundled together into one big file during building, I assume that one of them will end up overriding the other one. If, sometime in the future, these two images are no longer *identical* I suppose that that could lead to some problems.

Hence I'd suggest slightly tweaking the names to:
 - `--annotation-unfocused-field-background` in this file, for the new rule.
 - `--xfa--unfocused-field-background` in web/xfa_layer_builder.css, for the existing rule.